samples/subsys/ipc/openamp/sysbuild.cmake

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright 2022 NXP
+
+# Add external project
+ExternalZephyrProject_Add(
+    APPLICATION openamp_remote
+    SOURCE_DIR ${APP_DIR}/remote
+    BOARD ${SB_CONFIG_OPENAMP_REMOTE_BOARD}
+  )
+
+# Add a dependency so that the remote sample will be built and flashed first
+add_dependencies(openamp openamp_remote)
+# Place remote image first in the image list
```

Comment:
could you provide a bit more information as to why exactly this is required ?

For example like it's done for MCUboot:
https://github.com/zephyrproject-rtos/zephyr/blob/4a702ea21ef82b301464551807358f8490951e45/share/sysbuild/bootloader/CMakeLists.txt#L11-L12

Reply:
I've updated the comment- the build dependency exists because the remote image needs to generate a build header, using `CONFIG_BUILD_OUTPUT_INFO_HEADER`. The primary image includes that header, so the remote image must be built first.